### PR TITLE
Reduce more N+1 queries related to /api/v1/vulnerability collection.

### PR DESCRIPTION
**Pull request by bobmcwhirter**

### CODE REVIEW

File: modules/fundamental/src/vulnerability/model/mod.rs [hunk 1]
```diff
@@ -68,14 +68,18 @@ pub struct VulnerabilityHead {
...
-            .find_related(vulnerability_description::Entity)
-            .one(tx)
-            .await?
-            .filter(|model| model.lang == "en")
-            .map(|model| model.description);
+        let description = match description {
+            Some(inner) => inner.map(|inner| inner.description),
+            None => entity
+                .find_related(vulnerability_description::Entity)
+                .filter(vulnerability_description::Column::Lang.eq("en"))
```
Comment (JimFuller-RedHat) [inline modules/fundamental/src/vulnerability/model/mod.rs:78]: random observation (more for my education) ... how far has localisation been considered throughout this app ?

**Review by JimFuller-RedHat — APPROVED** [review]
as far as I could tell LGTM

File: modules/fundamental/src/vulnerability/model/mod.rs (hunk 1 above)
Comment (bobmcwhirter) [reply modules/fundamental/src/vulnerability/model/mod.rs:78]: The API doesn't have anything inherently in any language. We support multiple languages for descriptions, as the various formats we consume kinda do also.

l10n for the UI? No idea.  @carlosthe19916

File: modules/fundamental/src/vulnerability/model/mod.rs [hunk 2]
```diff
@@ -68,14 +68,18 @@ pub struct VulnerabilityHead {
 impl VulnerabilityHead {
     pub async fn from_vulnerability_entity(
         entity: &vulnerability::Model,
+        description: Option<Option<vulnerability_description::Model>>,
```
[the remark this answers is not in the document]
Comment (bobmcwhirter) [reply modules/fundamental/src/vulnerability/model/mod.rs:71]: Brilliant idea. Will do.

[the remark this answers is not in the document]
Comment (bobmcwhirter) [reply modules/fundamental/src/vulnerability/model/mod.rs]: Re-re, see how that looks to ya.

[the remark this answers is not in the document]
Comment (jcrossley3) [reply modules/fundamental/src/vulnerability/model/mod.rs]: MUCH CLEARER, thank you!

**Pull request merged**